cilium/cmd/bpf_ct_flush.go

```diff
@@ -27,7 +27,7 @@ import (
 
 // bpfCtFlushCmd represents the bpf_ct_flush command
 var bpfCtFlushCmd = &cobra.Command{
-	Use:    "flush ( <endpoint identifier> | global )",
+	Use: "flush ( <endpoint identifier> | global )",
```

Comment:
**[golangci]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
File is not `gofmt`-ed with `-s` (gofmt)